doc/parameters.md

```diff
@@ -478,6 +478,13 @@ connections.
 By default Gateway does not enable it, and the keepalive timeout on nginx is set
 to [75 seconds](http://nginx.org/en/docs/http/ngx_http_core_module.html#keepalive_timeout)
 
+### `APICAST_LUA_SOCKET_KEEPALIVE_REQUESTS`
+
+**Value:** positive integers
+**Example:** "1"
+
+Sets the maximum number of requests that can be served through one keepalive connection.
```

Comment:
Let's make it clear that it affects connections opened by apicast, so apicast is the client. This is not about the number of request being served by APIcast as a server before closing the connection.